docs/migration/5_0.md

```diff
@@ -41,13 +41,6 @@ under the section `headerDisplayOnly`.
 
 - `configurator.addToCart.buttonDisplayOnly` was added to `configurator-common.ts`
 
-### PaymentMethodsEffects
-
-- `GlobalMessageService` is a new, required constructor dependency.
-- `deleteUserPaymentMethod$` effect modified to trigger a global message
-
-#### CardComponent
```

Comment:
Not sure if this was intentional?
I see this section empty?

Reply:
@znikola I am not sure what you are referring to in this question. I think you mean like 49, the removal of #### CardComponent? 

If that is what you are referring to I removed because it was duplicate header and empty